source/configure/database-configuration-settings.rst

```diff
@@ -43,15 +43,34 @@ Data source
 | String input.                                                 | - Environment variable: ``MM_SQLSETTINGS_DATASOURCE``                    |
 |                                                               |                                                                          |
 +---------------------------------------------------------------+--------------------------------------------------------------------------+
-| To enable SSL:                                                                                                                           |
-|                                                                                                                                          |
-| - Add ``&tls=true`` to your database connection string if your SQL driver supports it.                                                   |
-| - Add ``&tls=skip-verify`` if you use self-signed certificates.                                                                          |
-+---------------------------------------------------------------+--------------------------------------------------------------------------+
 | **PostgreSQL databases**                                                                                                                 |
 |                                                                                                                                          |
 | When **Driver Name** is set to ``postgres``, use a connection string in the form of:                                                     |
 | ``postgres://mmuser:password@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10.``                                        |
+|                                                                                                                                          |
+| **To use TLS with PostgreSQL databases**:                                                                                                |
+|                                                                                                                                          |
+| The parameter to encrypt connection against a PostgreSQL database is ``sslmode`` and it authorize not all mode that you could define     |
```

Comment:
"it authorize not all mode that you could define with the CLI" - this sentence isn't easy to parse. 

When ``sslmode`` is used, is CLI functionality limited in some specific way?

Reply:
Application of the recommendations as suggested by @cwarnermm and @coltoneshaw